Runtime/Scripts/BaseMeshSync.Modifiers.cs

```diff
@@ -5,6 +5,7 @@
 using System.Text;
 using System.Threading.Tasks;
 using Unity.MeshSync;
+using UnityEditor;
```

Comment:
This script is under Runtime so this change should cause build issues. Why do you need the Editor here?

Reply:
Right, fixed